source/documentation/managing_apps/autoscaling.erb

```diff
@@ -0,0 +1,288 @@
+  * removing one instance if the average number of external HTTP requests per second per instance is less than 90 for 60 seconds
+
+This policy will ensure:
+
+  * there are never more than 6 instances of your app
+  * there are never fewer than 3 instances of your app
+
+This policy will avoid scaling multiple times within a 60 second period.
+
+#### Scale out during the day
+
+Imagine you have an app which is used frequently during daytime working hours,
+but does not receive much traffic otherwise.
+
+```
+{
+  "instance_min_count": 2,
+  "instance_max_count": 4,
+  "scaling_rules": [
+    {
+      "metric_type": "throughput",
+      "breach_duration_secs": 60,
+      "threshold": 90,
+      "operator": "<",
+      "cool_down_secs": 60,
+      "adjustment": "-1"
+    },
+    {
+      "metric_type": "throughput",
+      "breach_duration_secs": 60,
+      "threshold": 90,
+      "operator": ">=",
+      "cool_down_secs": 60,
+      "adjustment": "+1"
+    }
+  ],
+  "schedules": {
+    "timezone": "Europe/London",
+    "recurring_schedule": [
+      {
+        "start_time": "08:00",
+        "end_time": "18:00",
+        "days_of_week": [1, 2, 4, 5],
+        "instance_min_count": 4,
+        "instance_max_count": 8,
+        "initial_min_instance_count": 4
+      },
+      {
+        "start_time": "08:00",
+        "end_time": "18:00",
+        "days_of_week": [3],
+        "instance_min_count": 6,
+        "instance_max_count": 10,
+        "initial_min_instance_count": 6
+      }
+    ]
+  }
+}
+```
+
+This policy uses the same scaling rules as the previous example:
+
+  * adding one instance if the average number of external HTTP requests per second per instance is greater than 90 for 60 seconds
+  * removing one instance if the average number of external HTTP requests per second per instance is less than 90 for 60 seconds
+
+This policy will ensure:
+
+  * there are between 6 and 10 instances during Wednesday working hours
+  * there are between 4 and 8 instances during working hours on other weekdays
+  * there are between 2 and 4 instances otherwise
+
+#### Scale out during the week
+
+Imagine you have an app which is used frequently during the first week of the
+month, but does not receive much traffic otherwise.
+
+```
+{
+  "instance_min_count": 2,
+  "instance_max_count": 4,
+  "scaling_rules": [
+    {
+      "metric_type": "throughput",
+      "breach_duration_secs": 60,
+      "threshold": 90,
+      "operator": "<",
+      "cool_down_secs": 60,
+      "adjustment": "-1"
+    },
+    {
+      "metric_type": "throughput",
+      "breach_duration_secs": 60,
+      "threshold": 90,
+      "operator": ">=",
+      "cool_down_secs": 60,
+      "adjustment": "+1"
+    }
+  ],
+  "schedules": {
+    "timezone": "Europe/London",
+    "recurring_schedule": [
+      {
+        "start_time": "06:00",
+        "end_time": "22:00",
+        "days_of_month": [1, 2, 3, 4, 5, 6, 7],
```

Comment:
Should this be days of the week or days of the month?

Reply:
Resolved in e40f311